tox.ini

```diff
@@ -1,13 +1,23 @@
 # This file is part of REANA.
-# Copyright (C) 2022, 2023 CERN.
+# Copyright (C) 2022, 2023, 2024 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [tox]
-envlist = py38, py39, py310, py311, py312
+envlist =
+    py38
+    py39
+    py310
+    py311
+    py312
```

Comment:
Note: for Python 3.12 we need to first release new `pytest-reana` that is bumping the limit of the `coverage` dependency. This is done in https://github.com/reanahub/pytest-reana/pull/134